src/mantarray_file_manager/file_writer.py

```diff
@@ -156,3 +165,150 @@ def migrate_to_latest_version(
         current_file_path = migrate_to_next_version(
             current_file_path, working_directory=working_directory
         )
+
+
+def h5_file_trimmer(
+    file_path: str,
+    from_start: int,
+    from_end: int,
+) -> str:
+    """Trims an H5 file.
+
+    To use from the command line: `python -c "from mantarray_file_manager import h5_file_trimmer; h5_file_trimmer('tests/h5/v0.4.2/MA190190000__2021_01_19_011931__C3__v0.4.2.h5')"`
+
+    Args:
+        file_path: path to the H5 file
+        from_start: centimilliseconds to trim from the start
+        from_end: centimilliseconds to trim from the end
+
+    Returns:
+        The path to the trimmed H5 file. The amount actually trimmed off the file is dependent on the timepoints of the tissue sensor data and will be reflected in the new file name, message to the terminal, and the metadata. If the amount to be trimmed off is in between two time points, less time will be trimmed off and the lower timepoint will be used if from_start or upper timepoint if from_last. Reference sensor readings are trimmed according to the amount trimmed from tissue data.
+    """
+    # pylint: disable-msg=too-many-locals
```

Comment:
if you're disabling a pylint rule, generally we follow it with an additional comment justifying why we think it is OK in this case to ignore pylint. that way in the future (and during code reviews) people can agree or disagree with your reason.  `# pylint: disable-msg=too-many-locals # Anna (1/27/21): Some reason`